ros/openapi/openapi.json

```diff
@@ -72,6 +72,9 @@
                     },
                     {
                         "$ref": "#/components/parameters/filterStateParam"
+                    },
+                    {
+                        "$ref": "#/components/parameters/filterOSParam"
```

Comment:
Just an observation -
`Advisor` supports filter called `rhel_version` where it accepts version values example - 7.3, 8.5
whereas Patch > Systems supports filter called `os` where it accepts full version name example - RHEL 7.3,RHEL 8.5
Here, you are using filter name as `os` and accepts version values example - 7.3, 8.5

Secondly, on UI side both services are showing full name on filter chip whereas with PR https://github.com/RedHatInsights/ros-frontend/pull/110 change it shows just version value.


For reference attaching screenshots:

Advisor Systems -
![Screenshot from 2022-04-08 14-45-56](https://user-images.githubusercontent.com/6470528/162406653-bfe5c057-2aa0-4436-9b27-6c3123ba28dc.png)

Patch > Systems -
![Screenshot from 2022-04-08 14-47-07](https://user-images.githubusercontent.com/6470528/162406666-8e97c604-979d-41e0-8048-cbc06e6267b2.png)

Code functionality wise looks good but I would like to know opinions on which way we would like to keep this behavior?

Reply:
Since API response has field name as `os` so, I would prefer keeping it as `os` here as well. For the corresponding value, we can keep it as `RHEL x.y` but, my only query is what if user manually tries to hit the API and enters `RHELx.y` without a space? Keeping that in mind, I had kept these values as `x.y` and not `RHEL x.y`. Let me know if that is not a concern, I can update it to `RHEL x.y`.

Reply:
I agree with @kgaikwad it will be good to be consistent with other services and have param name as `os` with full values `RHEL x.y` and showing full name on the frontend side in the filter chip 👍🏻

Reply:
done.

Reply:
done.